tests/app/views/handlers/conftest.py

```diff
@@ -0,0 +1,33 @@
+from unittest.mock import Mock
```

Comment:
Consider using [monkeypatch](https://docs.pytest.org/en/6.2.x/monkeypatch.html), i will leave it up to you.

Reply:
Haven't actually used `monkeypatch` for test yet, but we could consider that, atm I used `Mock` as that is what we do across runner.